inconsistencies.md

```diff
@@ -0,0 +1,153 @@
+# Solving Cassandra consistency
+
+This plan makes the following assumption:
+
+  - We are not willing to rewrite mailbox/cassandra with event sourcing
+  - We will rather identify projections and how to heal them
+
+# Action plan
+
+(prioritized) this section present the conclusion of this studu which details can be found below.
+
+## Grooming
+
+Performance impact: lightweight transaction VS counters
+
+## High
+
+ - SolveInconsistencies webadmin tasks:
+   - messages (MessageIdToImapUid -> MessageIdTable)
+   - ACLs (ACL/CassandraUserMailboxRightsTable)
+   - Mailbox counters (/!\ counters)
+   - Current quota (/!\ counters)
+
+ - Testing:
+   - MessageIdToImapUid -> MessageIdTable
+   - Message <-> attachment
+   - Upload attachment
+
+ - Retry:
+   - messages (MessageIdToImapUid -> MessageIdTable)
+
+### Low
+
+Used to speed up IMAP operation (SELECT / EXPUNGE)
+
+ - SolveInconsistencies task:
+   - Applicale Flags
+   - Deleted Messages module
+   - First unseen
+   - Mailbox recent
+
+# Details
+
+## mailbox/mailboxPath
```

Comment:
Could we have a classical structured analysis:
* problem statement "tables X, used to get x from y, and table Y, used to get x from z, are written in the following ordering ... that could lead to entries written in ... but not ... The impacts are ..."
* solutions: "we can use ... as a source of truth to fix ... assuming ... limitations are ..."
* implementation: ...

Reply:
I will rework the form tommorrow if you wish.

Hopefully we'll have a debate on the merits